Source/Core/Core/Host.h

```diff
@@ -66,6 +66,8 @@ void Host_JitProfileDataWiped();
 void Host_UpdateMainFrame();
 void Host_UpdateTitle(const std::string& title);
 void Host_YieldToUI();
+void Host_UpdateWiimoteExtension(int controller, int extension);
+void Host_UpdateWiimoteMotionPlus(int controller, bool attached);
```

Comment:
Host functions are a somewhat crude way of handling callbacks. What do you think of instead exposing the `SubscribableSettingValue`s and having the UI code subscribe directly to them? Can the UI code just set and forget callbacks for all four Wii Remotes, or will there be problems if e.g. a Wii Remote changes from None to Emulated Wii Remote? Would exposing the `SubscribableSettingValue`s expose too many implementation details beyond the ability to subscribe?